api/go.mod

```diff
@@ -21,19 +21,23 @@ require (
 	github.com/hashicorp/go-secure-stdlib/strutil v0.1.2
 	github.com/hashicorp/hcl v1.0.0
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/stretchr/testify v1.7.0
```

Comment:
From what I can tell, [1.8.4 is the latest version](https://github.com/stretchr/testify/tags) -- is there a reason we didn't go for that one?

Reply:
This might be because my local cache is stall. I'll match whatever we use in go.mod in the repository's root directory.